activestorage/test/models/attachments_test.rb

```diff
@@ -136,16 +136,26 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
   end
 
   test "identify newly-attached, directly-uploaded blob" do
-    # Simulate a direct upload.
-    blob = create_blob_before_direct_upload(filename: "racecar.jpg", content_type: "application/octet-stream", byte_size: 1124062, checksum: "7GjDDNEQb4mzMzsW+MS0JQ==")
-    ActiveStorage::Blob.service.upload(blob.key, file_fixture("racecar.jpg").open)
+    blob = simulate_direct_upload_blob
```

Comment:
It’s relevant to this test and the new one below it that the directly-uploaded blob’s content type is `"application/octet-stream"`. They should both pass that value in as a parameter.